src/dal/questionsAPI.ts

```diff
@@ -1,11 +1,15 @@
 import {instance} from '@src/dal/instance';
-import {questionType} from '@customTypes/test-types';
+import {newQuestionType} from '@customTypes/test-types';
+import {addQuestionToTest} from '@customTypes/testsAPI-types';
 
 export const questionsAPI = {
   getQuestions(id: number) {
     return instance.get(`/quiz/${id}`);
   },
-  createQuestion(params: questionType) {
-    return instance.post('/questions', {params});
+  createQuestion(params: newQuestionType) {
+    return instance.post('/questions', params);
+  },
+  addQuestionToTest(params: addQuestionToTest) {
```

Comment:
maybe it should be in testAPI?

Reply:
why here in questionsAPI you have 2 endpoints `quiz ` and `questions `

Reply:
I got it, fixed it